test/remote/gateways/remote_fortis_test.rb

```diff
@@ -155,6 +179,14 @@ def test_storing_credit_card
     assert_success store
   end
 
+  def test_storing_credit_card_with_location_as_option
```

Comment:
is there another case where the `location_id` is required ? or maybe would be better to remove it from the fixtures ?

Reply:
You can have location_id, in your fixtures/gateway optionally, but if you don't want to, then fo the store operation yopu need to send it as aGSF